### DISCUSSION TITLE
Mono for solaris

### DISCUSSION THREAD

**Discussion by miteshsinha007** [mono/mono#20946] [General]
Hi, On the site only can see download for Window,Linux and MacOS. Any idea where I can get installs for Solaris and AIX pease ?

**Comment by lextm** [comment]
The installer for Solaris/AIX hasn't been published for years. You might refer to the legacy article to compile from source code, https://www.mono-project.com/docs/about-mono/supported-platforms/solaris/ But keep in mind the information inside is really old, so you might need extra work to compile everything. Other discussions (more recent) on compiling Mono on Solaris can be found from a search engine.
